barretenberg/cpp/src/barretenberg/goblin/utils.hpp

```diff
@@ -0,0 +1,101 @@
+#include "barretenberg/commitment_schemes/commitment_key.hpp"
+#include "barretenberg/eccvm/eccvm_composer.hpp"
+#include "barretenberg/goblin/goblin.hpp"
+#include "barretenberg/goblin/translation_evaluations.hpp"
+#include "barretenberg/proof_system/circuit_builder/eccvm/eccvm_circuit_builder.hpp"
+#include "barretenberg/proof_system/circuit_builder/goblin_ultra_circuit_builder.hpp"
+#include "barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp"
+#include "barretenberg/translator_vm/goblin_translator_composer.hpp"
```

Comment:
Charlie says often that any 'utils' naming is an antipattern and I somewhat agree. I think I agree in that often the exercise of thinking 'what if I didn't name this utils' can be fruitful, but disagree in that sometimes the code isn't all that important and you can name it whatever and move on for now. 

As an exercise here, I guess I'd call it mock_circuits.hpp instead

Reply:
Done